### ISSUE TITLE
Channel list swipe actions cut off on iPad

### ISSUE THREAD

**Issue by crayment** [GetStream/stream-chat-swiftui#279]
## What did you do?

1. Present chat in a `.fullScreenCover`
2. Swipe on channel list
3. Tap 3 dots
4. See image below:

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/78082/222000403-1db9b51f-05d3-436a-b22b-a8fd74e9cdaa.png">

## What did you expect to happen?

The actions should not be cut-off by the edge of the screen.

## What happened instead?

The actions were cut off by the edge of the screen.

## GetStream Environment
**GetStream Chat version:** 4.28.0
**GetStream Chat frameworks:** StreamChatSwiftUI
**iOS version:** 16.2
**Swift version:** 5.7
**Xcode version:** 14.2
**Device:** iPad Pro (11-inch) 4th generation

**Comment by martinmitrevski** [comment]
Hi @crayment,

Thanks for reporting. I've tried reproducing this, but it works well on my side. This is the code I've been using:
```
VStack {
    Button {
        shown = true
    } label: {
        Text("Tap")
    }
    .fullScreenCover(isPresented: $shown) {
        ChatChannelListView(
            viewFactory: DemoAppFactory.shared
        )
        .navigationViewStyle(.columns)
    }
}
```
Can you please share more details about your implementation - how are you presenting the `ChatChannelListView`?

Thanks,
Martin

**Comment by crayment** [comment]
Thanks for the response! I did some more digging last night but forgot to follow up here. This seems to be related to using `embedInNavigationView: false` on `ChatChannelListView`. It's reproducible in the demo app with the change below:

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/78082/222198125-71037b42-dc9e-45d6-88cd-b4dcfedfb863.png">

I have worked around this for now by switching `embedInNavigationView` back to `true`.
